src/components/KpiGrid.vue

```diff
@@ -76,33 +80,30 @@ export default {
   height: 100%;
```

Comment:
Lurer på om `justify-content: space-between;` må tas ut nå, for å unngå at ting flytter på seg når man har noen kort med verdi, andre uten.